gbpservice/nfp/orchestrator/config_drivers/heat_driver.py

```diff
@@ -246,6 +246,20 @@ def loadbalancer_post_stack_create(self, network_function_details):
                     if port_info['port_model'] != nfp_constants.GBP_PORT:
                         return
 
+    def _post_stack_create(self, nfp_context):
+        service_details = self.get_service_details_from_nfp_context(
+            nfp_context)
+        service_details = service_details['service_details']
+        service_type = service_details['service_type']
+
+        if service_type in [pconst.LOADBALANCER]:
+            logging_context = nfp_logging.get_logging_context()
+            auth_token = logging_context['auth_token']
+            provider_tenant_id = nfp_context['tenant_id']
+            provider = service_details['provider_ptg']
+            self._create_policy_target_for_vip(
```

Comment:
In **neutron/services/grouppolicy/plugin.py** in delete_policy_target_group api,
 554         with session.begin(subtransactions=True):
 555             for pt in self.get_policy_targets(context, {'id': pt_ids}):
 556                 # We will allow PTG deletion if all PTs are unused.
 557                 # We could have cleaned these opportunistically in
 558                 # the previous loop, but we will keep it simple,
 559                 # such that either all unused PTs are deleted
 560                 # or nothing is.
 561                 self.delete_policy_target(context, pt['id'])
 562             super(GroupPolicyPlugin, self).delete_policy_target_group(
 563                 context, policy_target_group_id)
 564
 565         try:
 566             self.policy_driver_manager.delete_policy_target_group_postcommit(
 567                 policy_context)

gbp is deleting all unused pt's.